### PR TITLE
Fix/v6 semgrep findings

### PR DESCRIPTION
**Pull request by Sakchai-Refinitiv**
## Description

Fix Semgrep findings to keep EF maintaining code hygiene and to ensure apps adopting EF security compliant.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

JIRA: [ELF-2232](https://jira.refinitiv.com/browse/ELF-2232)

### CODE REVIEW

**Comment by bualoy-napat** [comment]
LGTM!

**Review by bualoy-napat — DISMISSED** [review]

**Review by bualoy-napat — DISMISSED** [review]

File: .github/actions/run-test/action.yml [hunk 1]
```diff
@@ -42,14 +42,14 @@ runs:
   steps:
     - name: 'BrowserStack Environment Setup' # Invokes the setup-env action
       if: ${{ inputs.browserstack == 'true' }}
-      uses: browserstack/github-actions/setup-env@master
+      uses: browserstack/github-actions/setup-env@0496628aa0145adb845337f6f84548256fe10eeb
```
Comment (wattachai-lseg) [inline .github/actions/run-test/action.yml]: missing branch ref. Is it `master`?

File: .github/actions/run-test/action.yml [hunk 2]
```diff
@@ -42,14 +42,14 @@ runs:
...
       with:
         username: ${{ inputs.username }}
         access-key: ${{ inputs.access-key }}
 
     - name: 'BrowserStack Local Tunnel Setup' # Invokes the setup-local action
       if: ${{ inputs.browserstack == 'true' }}
-      uses: browserstack/github-actions/setup-local@master
+      uses: browserstack/github-actions/setup-local@0496628aa0145adb845337f6f84548256fe10eeb
```
Comment (wattachai-lseg) [inline .github/actions/run-test/action.yml]: missing branch ref. Is it `master`?

File: .github/actions/run-test/action.yml [hunk 3]
```diff
@@ -88,6 +88,6 @@ runs:
 
     - name: BrowserStackLocal Stop # Terminating the BrowserStackLocal tunnel connection
       if: ${{ inputs.browserstack == 'true' }}
-      uses: browserstack/github-actions/setup-local@master
+      uses: browserstack/github-actions/setup-local@0496628aa0145adb845337f6f84548256fe10eeb
```
Comment (wattachai-lseg) [inline .github/actions/run-test/action.yml]: missing branch ref. Is it `master`?

File: .github/actions/run-test/action.yml (hunk 1 above)
Comment (Sakchai-Refinitiv) [reply .github/actions/run-test/action.yml]: Changed to v1.0.1

File: .github/actions/run-test/action.yml (hunk 2 above)
Comment (Sakchai-Refinitiv) [reply .github/actions/run-test/action.yml]: Changed to v1.0.1

File: .github/actions/run-test/action.yml (hunk 3 above)
Comment (Sakchai-Refinitiv) [reply .github/actions/run-test/action.yml]: Changed to v1.0.1

**Review by wattachai-lseg — DISMISSED** [review]
LGTM. Just need to verify result on semgrep dashboard.

**Review by wattachai-lseg — APPROVED** [review]

**Pull request merged**